airbyte-commons-temporal/src/main/java/io/airbyte/commons/temporal/TemporalClient.java

```diff
@@ -372,7 +373,11 @@ public TemporalResponse<ConnectorJobOutput> submitDiscoverSchema(final UUID jobI
         () -> getWorkflowStub(DiscoverCatalogWorkflow.class, TemporalJobType.DISCOVER_SCHEMA).run(jobRunConfig, launcherConfig, input));
   }
 
-  public TemporalResponse<StandardSyncOutput> submitSync(final long jobId, final int attempt, final JobSyncConfig config, final UUID connectionId) {
+  public TemporalResponse<StandardSyncOutput> submitSync(final long jobId,
+                                                         final int attempt,
+                                                         final JobSyncConfig config,
+                                                         final AttemptSyncConfig attemptConfig,
+                                                         final UUID connectionId) {
```

Comment:
I made the necessary changes here because we're building a sync input, but I couldn't really find where the temporal client's submitSync method actually runs. To me it seems like in practice we're using the GenerateInputActivity to do this.

Maybe @benmoriceau knows more about why this is here?

Reply:
Notably, this sync input is missing other recent additions, like source/destination actor IDs and allowed hosts

Reply:
I think that it is some legacy from before we had the ConnectionManagerWorkflow was not existing.
The WorkerRun (that call that method upon creation) seems to be only useful to get the jobRoot...

This type of starting a temporal job is still useful for check connection and discover schema.